### DISCUSSION TITLE
Stuck PR checks

### DISCUSSION THREAD

**Discussion by Felixoid** [community/community#25966] [Actions]
<p>Hello. Few commits ago I’ve renamed <code>build</code> step into <code>tests</code> <a href="https://github.com/Felixoid/clickhouse-plantuml/commit/0ea3794a841098f6252269f0cacf73c74298cfff#diff-353995fb5bebaef307ec25fa5be39042" rel="nofollow noopener">here</a>. Unfortunately, PRs still <a href="https://github.com/Felixoid/clickhouse-plantuml/pull/7" rel="nofollow noopener">require</a> a <code>build</code> matrix for some reason. I can’t find how to unlink them and, by logic, they should be gone.</p>
<p>Could you help me, please?</p>

**Comment by ghost** [comment, marked as answer]
<p>Hi, could you check if you have a <code>build</code> matrix as a required status check?</p>
<p><a href="https://docs.github.com/en/github/administering-a-repository/enabling-required-status-checks" class="onebox" target="_blank">https://docs.github.com/en/github/administering-a-repository/enabling-required-status-checks</a></p>

**Comment by Felixoid** [comment]
<p>Yes, it’s indeed there. Thank you very much</p>
